lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -1,32 +1,37 @@
 define((require, exports, module) ->
+  getContext = require("ace/ext/papeeria/latex_parsing_context").getContext
   exports.setupPreviewer = (editor, popoverHandler) ->
     katex = null
-    popoverHandler = popoverHandler ? {
+    popoverHandler ?= new class
```

Comment:
I don't think that you need it ti be a class instance. Classes and fat arrow bindings don't come for free, they bloat the code and runtime, so don't use them unless you  really need (e.g. if you create many instances)